src/config-as-code/repo-config-service.ts

```diff
@@ -0,0 +1,55 @@
+import YAML from "yaml";
+import GitHubClient from "../github/client/github-client";
+import { getCloudInstallationId } from "../github/client/installation-id";
+import { getLogger } from "../config/logger";
+import { RepoConfig } from "./repo-config";
+import RepoConfigDatabaseModel from "./repo-config-database-model";
+
+const logger = getLogger("services.config-as-code");
+const CONFIG_PATH = "aconfig.yml";
+
+const getRepoConfig = async (githubInstallationId: number, owner: string, repo: string): Promise<string | null> => {
+	const client = new GitHubClient(getCloudInstallationId(githubInstallationId), logger);
+	try {
+		const response = await client.getRepositoryContent(owner, repo, CONFIG_PATH)
+		const contentString = Buffer.from(response.data.content, "base64").toString("ascii"); 
+		logger.debug(`Converted ${response.data.content} to ${contentString}`);
+		return contentString;
+	} catch (err) {
```

Comment:
I think we should just let this error bubble up in the call stack. The caller should care about the proper error handling. Same with other catch blocks in the other functions.